apps/fbis-ccf/behaviours/submit.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const config = require('../../../config');
+const sendEmail = require('../../../lib/utils').sendEmail;
+const fields = require('../fields/index');
+const uuidv4 = require('uuid').v4;
+
+module.exports = superclass => class Submit extends superclass {
+
+  saveValues(req, res, next) {
+    let emailData = Object.keys(fields).reduce((data, field) => {
+      data[field] = req.form.historicalValues[field] || 'n/a';
+      return data;
+    }, {});
```

Comment:
learner question, how does the historicalValues work, is this the thing that was added by Andy Moody to HOF?

Reply:
Yeah, Andy added it. For whatever reason, `values` is empty at this stage but `historicalValues` contains all form fields and anything else that's been added to the session.